howto/monitor/cos.md

```diff
@@ -0,0 +1,8 @@
+Anbox Cloud uses [Canonical Observability Stack(COS)](https://charmhub.io/topics/canonical-observability-stack) to enable users to collect and monitor their workloads.
+
+Since Anbox Cloud uses Juju for deployment, by supporting [COS Lite](https://charmhub.io/topics/canonical-observability-stack/editions/lite) integration, we can export observability information with Prometheus, Loki and Grafana.
+
+Currently, only the [AMS charm](https://charmhub.io/ams) supports COS Lite integration. To gather observability metrics for the AMS charm,
```

Comment:
We have to be careful here as this is only true for an unreleased version of the charm which we do not officially support yet. We need to call this out here.